metadata/contributors.json

```diff
@@ -3025,6 +3025,18 @@
          "hta"
       ]
    },
+   {
+      "emails" : [
+         "hmw@apple.com"
+      ],
+      "expertise" : "Safari Technology Preview",
+      "github" : "harrisonweinerman",
+      "name" : "Harrison Weinerman",
+      "nicks" : [
+         "hmw"
+      ],
+      "status" : "committer"
```

Comment:
I don't think, per https://webkit.org/commit-and-review-policy/#criteria-for-committers, you are a committer.